browser/agent-browser/src/thread-manager.ts

```
@@ -11,6 +11,7 @@ import { BrowserManager } from 'agent-browser';
 import type { BrowserLaunchOptions } from 'agent-browser';
 import type { Page } from 'playwright-core';
 import type { BrowserConfig } from './types';
+import { installPlaywrightLinuxDeps } from './playwright-deps';
```

ESLint

[error] `./playwright-deps` import should occur before type import of `./types`

(import/order)